### ISSUE TITLE
intersect function in module three

### ISSUE THREAD

**Issue by empeirikos** [vectorgraphics/asymptote#367]
I am trying to use the `intersect` function for calculating the points of intersection between a plane and a sphere. 
I define a plane using,
```
triple u=(0,2,1);
triple v=(1,0,2);
triple A=(1,1,1);
pl=plane(u, v, A);
```
and a sphere using,
```
surface s=shift(A+(u+v)/2)*unitsphere;
```
When I write,
```
real[ ] ips;
ips = intersect(pl, s);
```

I get,

/usr/local/Cellar/asymptote/2.83/share/asymptote/three_surface.asy: 1296.25: Intersection of path3 with Bezier triangle is not yet implemented

yet this function is listed in the documentation:
<img width="782" alt="Screenshot 2023-01-31 at 22 10 36" src="https://user-images.githubusercontent.com/80323854/215895271-c0b22892-9cb9-49a3-abda-650131edd487.png">

**Comment by johncbowman** [comment]
First of all, your code doesn't even compile. Here is what you intended (to save me time, it is better to post complete tested code, not snippets):

```
import three;

triple u=(0,2,1);
triple v=(1,0,2);
triple A=(1,1,1);
path3 pl=plane(u,v,A);

surface s=shift(A+(u+v)/2)*unitsphere;

real[ ] ips;
ips = intersect(pl,s);
picture pic;
size(pic,10cm);
fill(circle(0,1),red);

add(bbox(pic,1cm,NoFill));
```
At the time  `real[] intersect(path3 p, surface s, real fuzz=-1);` was documented, bezier triangles had not yet been implemented. No one has needed this feature until now, but I will look into it.

**Comment by empeirikos** [comment]
Apologies  for not including the preamble. The code I wrote originally did not include the `intersect` command and compiled without any errors. Then after I included the lines 
```
real[ ] ips;
ips = intersect(pl,s);
```
I got the error message. I will make sure I include the complete tested code in any new queries. By the way if there is another way to calculate a spherical cross-section I will avoid using the `intersect` command.
